apps/web-giddh/webpack.partial.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
             'errlyticsNeeded': JSON.stringify(false),
             'errlyticsKey': JSON.stringify(''),
             'AppUrl': JSON.stringify('http://localhost:3000/'),
-            'ApiUrl': JSON.stringify('https://apitest.giddh.com/'),
+            'ApiUrl': JSON.stringify('http://giddh-template.eu-west-1.elasticbeanstalk.com/'),
```

Comment:
Revert this change as it replaces ApiUrl to template URL so all the service calls that depend on this variable will use incorrect URL